ofApp.cpp

```diff
@@ -1,387 +0,0 @@
-        font2.drawString(wrappedTextTeaCup, 50, 200);
-        
-        choice1.set(200, 400, 250, 175); // where you see 4 numbers like this they generally represent xPos, yPos, width and height relative to (0,0), which is the top left corner
-        choice2.set(700, 400, 250, 175);
-        
-        ofDrawRectangle(choice1);
-        ofDrawRectangle(choice2);
-
-        populationButton.draw(200, 400, 250, 175);
-        areaButton.draw(700, 400, 250, 175);
-    }
-    else if (screen == screens::game) {
-        backgroundImage.draw(0, 0, 1200, 750); // this should cover the whole game screen at all times
-
-        leftOptionImage.draw(0, 0, 600, 750);
-        font.drawString(wrapString(leftObj.jsonStringName, 550, true), 50, 65);
-        //ofDrawBitmapString(leftObj.population, 20, 500); //YzxNkP99T9WYb8
-        //ofDrawBitmapString(leftObj.area, 20, 525); //YzxNkP99T9WYb8
-        leftObj.jsonImg.draw(0, 130, 600, 300); //this does not work since not all flags are the same shape - for example switzerland's flag is square not rectangular
-
-        rightOptionImage.draw(600, 0, 600, 750);
-        font.drawString(wrapString(rightObj.jsonStringName, 550, true), 650, 65);
-        //ofDrawBitmapString(rightObj.population, 620, 500); //YzxNkP99T9WYb8
-        //ofDrawBitmapString(rightObj.area, 620, 525); //YzxNkP99T9WYb8
-        rightObj.jsonImg.draw(600, 130, 600, 300); 
-
-
-        leftOptionButton.set(0, 0, 600, 750);
-        rightOptionButton.set(600, 0, 600, 750);
-
-        // score drawn last as it is going to cover both sides of the screen and this has a first come first served basis for drawing. whatever is drawn first ends up on the bottom. and by bottom i OF COURSE mean farthest away from the camera on the z-axis
-        font.drawString("Score:", 450, 550); // will not need to wrap the score since ppl unlikely to get a score worthy of being wrapped
-        font.drawString(ofToString(leftObj.score), 640, 550); // need ofToString to convert it quickly to a string so it is valid - found that here - https://openframeworks.cc/documentation/graphics/ofGraphics/#show_ofToString
-
-        ofSetColor(208, 72, 72);
-
-        if (leftObj.colour != "left") { // if the mouse is hovering over your option, then redraw the name of said option in a different colour - thats all this is doing but it provides a nice affect
-            font.drawString(wrapString(leftObj.jsonStringName, 550, true), 50, 65);
-        }
-        if (rightObj.colour != "right") {
-            font.drawString(wrapString(rightObj.jsonStringName, 550, true), 650, 65);
-        }
-        ofSetColor(255); // reset colour back to white - likely dont need this as it is also at the start of the draw function but may cause issues when using keys to move between screens if removed
```

Comment:
Have you tested this? should be a quick one to test and see if this could be removed. If you have this at the start of the draw() function it should reset every frame anyway